src/App.tsx

```diff
@@ -1,61 +1,119 @@
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { useState } from 'react';
+import { FormData, Todo } from './types';
+import { TodoList } from './components/TodoList';
+import { getNewTodoId } from './services';
+
+const initialForm: FormData = {
+  title: '',
+  userId: 0,
+};
 
 export const App = () => {
+  const [todos, setTodos] = useState<Todo[]>(todosFromServer);
+  const [formData, setFormData] = useState(initialForm);
+  const [isTitleError, setIsTitleError] = useState(false);
+  const [isUserError, setIsUserError] = useState(false);
+
+  const { title, userId } = formData;
+
+  const isDisabledAdd = !title.trim() || !userId;
+
+  const reset = () => {
+    setFormData({ ...initialForm });
+  };
+
+  const handleAddTodo = (inputData: FormData) => {
+    const newTodo = {
+      ...inputData,
+      id: getNewTodoId(todos),
+      completed: false,
+    };
+
+    setTodos(currentTodos => [...currentTodos, newTodo]);
+  };
```

Comment:
The `handleAddTodo` function correctly adds a new todo with a unique ID and default `completed` status. Ensure that `getNewTodoId` is implemented correctly to avoid ID conflicts.